packages/storybook/src/theme.ts

```diff
@@ -439,6 +439,16 @@ const navs = {
   },
 }
 
+const progresses = {
+  container: {
```

Comment:
nice, I didn't like track/rail (and train 🚂  /track seemed to stretch that metaphor too far) but this makes sense